src/SCRIPTS/BF/MSP/ghost.lua

```diff
@@ -0,0 +1,28 @@
+-- GHOST Frame Types
+local GHST_FRAMETYPE_MSP_REQ           = 0x21
+local GHST_FRAMETYPE_MSP_WRITE         = 0x22
+local GHST_FRAMETYPE_MSP_RESP          = 0x28
```

Comment:
Can we have fewer spaces here? 
I see that were calling it both "ghst" and "ghost". Which one is the "correct" one?

Reply:
I prefer `ghst` as it goes nice with `csrf`

Reply:
Fixed